core/controllers/profile_test.py

```diff
@@ -312,3 +312,60 @@ def test_contribution_datetime(self):
         self.assertEqual(
             response_dict['first_contribution_datetime'],
             utils.get_time_in_millisecs(current_datetime))
+
+
+class UserContributionsTests(test_utils.GenericTestBase):
+
+    USERNAME_A = 'a'
+    EMAIL_A = 'a@example.com'
+    USERNAME_B = 'b'
+    EMAIL_B = 'b@example.com'
+    EXP_ID_1 = 'exp_id_1'
+
+    def test_zero_count(self):
+        #Check to see a user with no contributions
+        #shows up as 0 created and edited on the profile page.
```

Comment:
The grammar here is a bit off. How about:

    Check that the profile page for a user with no contributions shows
    that they have 0 created/edited explorations.

I.e., say what 'created/edited' refers to. Ditto for the other comments.